components/brave_webtorrent/extension/components/mediaViewer.tsx

```diff
@@ -60,4 +60,20 @@ export default class MediaViewer extends React.PureComponent<Props, {}> {
 
     return <div className='mediaViewer'>{content}</div>
   }
+
+  componentDidMount () {
+    // Make our video "full screen" so it looks nice.
+    // This and CSS give it a letterbox effect.
+    if (document.getElementById('video') !== null) {
+      let style = document.body.style
+      style.paddingTop = '0px'
+      style.paddingLeft = '0px'
+      style.paddingBottom = '0px'
+      style.paddingRight = '0px'
+      style.marginTop = '0px'
+      style.marginLeft = '0px'
+      style.marginBottom = '0px'
+      style.marginRight = '0px'
```

Comment:
Is there a reason why these styles cannot be applied using CSS?

```css
video {
  padding: 0;
  margin: 0;
}
```

Reply:
To be honest it's been so long ago now I don't quite remember if there is a reason or it was just something I overlooked.